autopush/websocket.py

```diff
@@ -352,13 +360,31 @@ def finish_hello(self, result):
             self.sendMessage(json.dumps(msg).encode('utf8'), False)
             return
 
+        if result.get("Attributes", {}).get("node_id"):
+            # Get the previous information returned from dynamodb.
+            attrs = result.get("Attributes")
+            node_id = self._get_aval(attrs.get("node_id", {}), 'S')
+            uaid = self._get_aval(attrs.get("uaid", {}), 'S')
+            connected_at = self._get_aval(attrs.get("connected_at", {}), 'N')
+            url = "%s/notif/%s/%s" % (node_id, uaid, connected_at)
+            if not getattr(self, 'testing', False):
+                d = self.ap_settings.agent.request(
+                    "DELETE",
+                    url.encode("utf8"),
+                ).addCallback(self.finish_hello)
```

Comment:
My main worry is if that other node is gone, we wait upwards of 10 seconds for this DELETE, and the client times out waiting for us and reconnects. We have to respond to these commands quickly. It can be deferred, we just don't need to wait for it.